docs/registry/index.rst

```diff
@@ -602,7 +602,6 @@ RegTAP services using:
   ...   for r in res)))
   http://dc.zah.uni-heidelberg.de/tap
   http://gavo.aip.de/tap
-  http://voparis-rr.obspm.fr/tap
```

Comment:
That paris fell out here was an accident.  It's back again, so please undo this edit.